plugins/modules/net_tools/nmcli.py

```diff
@@ -342,6 +342,13 @@
             - Name of the Wireless router or the access point.
        type: str
        version_added: 3.0.0
+    wifi_mode:
+        description:
+            - Wi-Fi network mode.
+        default: infrastructure
```

Comment:
```suggestion
```
I would not default this option as nmcli already does this. (See [here](https://developer.gnome.org/NetworkManager/stable/settings-802-11-wireless.html))

Reply:
Yep, I noticed that but wasn't sure the best approach here, I'll take this out. Thanks!